package.json

```
@@ -25,6 +25,6 @@
   "license": "MIT",
   "peerDependencies": {
     "@svgr/webpack": ">=2.0.0",
-    "gatsby": ">=2.0.0"
+    "gatsby": ">=3.0.0"
```

```suggestion
    "gatsby": "^3.0.0"
```

Hi, Gatsby maintainer here! I'd suggest using the caret here as otherwise any future majors would be automatically compatible with this. It might be the case that that's true but as we saw with the previous `>=2.0.0` range we had some v3 breaking changes (webpack 5) for this package.

We want to display compatibility of a plugin on the `/plugins` page in the future and we'll rely on the `peerDep` there so we'll need to have this more explicit :)